website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -0,0 +1,118 @@
+---
+
+Much of this documentation is also found at [Cumulus](https://github.com/nasa/cumulus/blob/master/docs/development/release.md).
+
+## Versioning
+
+The ORCA team uses semantic versioning. More information about semantic
+versioning can be found [here](https://semver.org/).
+
+## Release Process
+
+1. **Create a release branch.**
+
+From develop, create a new release branch from develop following the
+`release-MAJOR.MINOR.x`. For example, `release-1.14.1`. Push this branch 
+to github if you created it locally.
+
+2. **Update CHANGELOG.md.**
+
+Update the [CHANGELOG.md](https://github.com/nasa/cumulus-orca/blob/master/CHANGELOG.md). 
+Put a header under the 'Unreleased' section with the new version number and 
+the date.
+
+3. **Create a git tag for the release.**
+
+Make sure you're on the latest commit of the release branch.
+
+Create and push a new git tag:
+
+```
+    git tag -a vx.y.z -m "Release x.y.z"
+    git push origin vx.y.z
+```
+
+4. **Running the deployment**
+
+Publishing of new releases is handled by a Bamboo release plan and is manually
+triggered.
+
+If you created a new release plan in step one, you will need to create a new
+bamboo deployment plan.
+
+**Creating a Bamboo deployment plan**
+
+* In the ORCA project (https://ci.earthdata.nasa.gov/browse/ORCA-OI), click
+  `Actions -> Configure Plan` at the top right.
+* Scroll to the bottom of the branch list in the bottom left and select
+  `Create Plan Branch`.
+* Add the values in that list. Choose a display name that makes it very clear 
+  this is a deployment branch plan. Release (branch name) seems to work well. 
+  Make sure you enter the correct branch name.
+* Important Deselect Enable Branch - if you do not do this, it will immediately
+  fire off a build.
+* Do Immediately On the Branch Details page, enable 'Change Trigger'. Set the 
+  Trigger type to manual, and this will prevent commits to the branch from 
+  triggering the build plan. You should have been redirected to the 'Branch 
+  Details' tab after creating the plan. If not, navigate to the branch from
+  the list where you clicked 'Create Plan Branch' in the previous step.
+* Go to the Variables tab. Ensure that you are on your branch plan and not the
+  master plan: You should not see a large list of configured variables, but 
+  instead a dropdown allowing you to select variables to override, and the tab 
+  title will be Branch Variables. Set a DEPLOYMENT variable appropriate for the
+  release (defaults to last committer). This should be cumulus-from-npm-tf 
+  except in special cases such as incompatible backport branches. Then set:
+  * ORCA_VERSION: `<version number>`
+  * RELEASE_FLAG: true
+* Enable the branch from the 'Branch Details' page.
+* Run the branch using the 'Run' button in the top right.
+
+Bamboo will build and run unit tests against that tagged release.
+
+**Create a new ORCA release on github**

```

Comment:
This should be an H2 header `##` instead of bolded.

Reply:
Made into an H2 header.